playbook/app/pb_kits/playbook/pb_filter/Filter/index.tsx

```diff
@@ -1,33 +1,59 @@
-import React from 'react'
-import FilterSingle, { FilterSingleProps } from './FilterSingle'
-import FilterDouble, { FilterDoubleProps } from './FilterDouble'
+import React, { useState } from 'react';
+import FilterSingle, { FilterSingleProps } from './FilterSingle';
+import FilterDouble, { FilterDoubleProps } from './FilterDouble';
 
 type FilterProps =
   | FilterSingleProps
   | (FilterDoubleProps & {
-      double?: boolean,
-    })
+      double?: boolean;
+    });
 
 const Filter = ({
   double = false,
   ...templateProps
-  }: FilterProps): React.ReactElement => {
+}: FilterProps): React.ReactElement => {
+  const [selectedDate, setSelectedDate] = useState<string | null>(null);
```

Comment:
This is where im grabbing what is selected on the date picker kit and setting state with the value